rubrics/api/src/main/bundle/rubrics.properties

```diff
@@ -21,6 +21,7 @@ save=Save
 done=Done
 total=Total
 confirm_remove=Are you sure you want to remove "{}" ?
+confirm_remove_public=Are you sure you want to remove "{}" both from publicly shared rubrics and from its site origin?
```

Comment:
I'm not sure about the word "public". This implies that these rubrics are visible to people who are not Sakai users, the public.